enterprise/reporting/error_reporting.go

```diff
@@ -581,6 +581,12 @@ func (edr *ErrorDetailReporter) sendMetric(ctx context.Context, label string, me
 }
 
 func (edr *ErrorDetailReporter) Stop() {
+	edr.syncersMu.RLock()
+	for _, syncer := range edr.syncers {
+		_ = syncer.DbHandle.Close()
+	}
+	edr.syncersMu.RUnlock()
+
```

Comment:
`DatabaseSyncer` method creates the handle, it should also be the one that needs to close it